### ISSUE TITLE
trim unnecessary spaces

### ISSUE THREAD

**Issue by akira6592** [akira6592/vscode-l2mdtable#6]
```markdown
- 
  - aaa
  - bbb
  -    ccc
- 
  - aaa
  - bbb
  -    ccc
```

should be

```markdown
|aaa|bbb|ccc|
|:------|:------|:------|
|aaa|bbb|ccc|
```

actually

```markdown
|aaa|bbb|   ccc|
|:------|:------|:------|
|aaa|bbb|   ccc|
```

**Comment by akira6592** [comment]
```python
let matchedCol = flatStringArray[i].match(/^ +- (.*)/);
```
I'll modified like below
```python
let matchedCol = flatStringArray[i].match(/^ +- +(.*)/);
```

**Comment by akira6592** [comment]
fixed by v0.1.0
